README.md

```diff
@@ -169,31 +172,43 @@ For faster production-tier searches, Qdrant also provides a gRPC interface. You
 
 ### Filtering and Payload
 
-Qdrant enables JSON payloads to be associated with vectors, providing both storage and filtering based on payload values. It supports various combinations of `should`, `must`, and `must_not` conditions, ensuring retrieval of all relevant vectors unlike `ElasticSearch` post-filtering.
+Qdrant can attach any JSON payloads to vectors, allowing for both the storage and filtering of data based on the values in these payloads.
+Payload supports a wide range of data types and query conditions, including keyword matching, full-text filtering, numerical ranges, geo-locations, and more.
```

Comment:
I'd prefer not using the "full-text filtering" since readers coming from Elastic might confuse it with full-text search
```suggestion
Payload supports a wide range of data types and query conditions, including keyword matching, exact text filtering, numerical ranges, geo-locations, and more.
```

Reply:
it was about full-text search, actually.